### PR TITLE
Fetch start of epoch metablock

### PR DESCRIPTION
**Pull request by ssd04**

### CODE REVIEW

File: cmd/proxy/config/apiConfig/v1_0.toml [hunk 1]
```diff
@@ -113,6 +113,6 @@ Routes = [
     { Name = "/:shard/raw/block/by-hash/:hash", Secured = false, Open = true, RateLimit = 0 },
     { Name = "/:shard/json/block/by-nonce/:nonce", Secured = false, Open = true, RateLimit = 0 },
     { Name = "/:shard/json/block/by-hash/:hash", Secured = false, Open = true, RateLimit = 0 },
-    { Name = "/:shard/raw/miniblock/by-hash/:hash", Secured = false, Open = true, RateLimit = 0 },
-    { Name = "/:shard/json/miniblock/by-hash/:hash", Secured = false, Open = true, RateLimit = 0 }
+    { Name = "/:shard/raw/miniblock/by-hash/:hash/epoch/:epoch", Secured = false, Open = true, RateLimit = 0 },
+    { Name = "/:shard/json/miniblock/by-hash/:hash/epoch/:epoch", Secured = false, Open = true, RateLimit = 0 }
```
Comment (bogdan-rosianu) [inline cmd/proxy/config/apiConfig/v1_0.toml:117]: forgotten startofepoch endpoints?

File: process/blockProcessor.go [hunk 2]
```diff
@@ -20,7 +20,9 @@ const (
 	internalMetaBlockByNoncePath  = "/internal/%s/metablock/by-nonce"
 	internalShardBlockByNoncePath = "/internal/%s/shardblock/by-nonce"
 
-	internalMiniBlockByHashPath = "/internal/%s/miniblock/by-hash"
+	internalMiniBlockByHashPath = "/internal/%s/miniblock/by-hash/%s/epoch"
+
+	internalStartOfEpochMetaBlockPath = "/internal/%s/startofepoch/metablock/by-epoch"
```
Comment (bogdan-rosianu) [inline process/blockProcessor.go:25]: can we add the epoch param in these strings as well?

File: cmd/proxy/config/apiConfig/v1_0.toml (hunk 1 above)
Comment (ssd04) [reply cmd/proxy/config/apiConfig/v1_0.toml]: right. added them

File: process/blockProcessor.go (hunk 2 above)
Comment (ssd04) [reply process/blockProcessor.go]: done :+1:

File: cmd/proxy/config/apiConfig/v1_0.toml (hunk 1 above)
Comment (iulianpascalau) [reply cmd/proxy/config/apiConfig/v1_0.toml:117]: Forgotten `,` ? This looks like an invalid .toml file
Also, cmd/proxy/config/apiConfig/v_next.toml

**Review by iulianpascalau — CHANGES_REQUESTED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**